src/store/modules/RequestsModule.js

```diff
@@ -21,10 +23,8 @@ class RequestsModule extends VuexModule {
 
   @Action
   async getNextNonce(address) {
-    const web3 = await signer.getWeb3Instance();
-    const nonce = await web3.eth.getTransactionCount(address);
-
-    return nonce;
+    const nonce = await web3.call('eth_getTransactionCount', address);
```

Comment:
So, what do you think, @kazurus, @dudiq and @ifiri, about having "sugar" methods for each method in `web3`? Like `web3.getTransactionCount`? I guess it should not make library bloat as original `web3`.

Reply:
We have already discussed, will add